src/components/UseDialogCard.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <div id='page-content'>
+    <div id='svg-container'>
+    </div>
+    <div id="dialog-card-setup">
+      <dialogCard 
+          :show="showDialog"
+          :header="cardHeader"
+          :title="cardTitle"
+          :text="cardText"
+          :source="cardImageSource"
+          :altText="cardAltText"
+          :close="close"
+        />
+    </div>
+  </div>
+</template>
+
+<script>
+import * as d3Base from 'd3';
+export default {
+    name: 'useDialogCard',
+    components: {
+      dialogCard: () => import( /* webpackPreload: true */ /*webpackChunkName: "section"*/ "./../components/Dialog")
+    },
+    data() {
+      return {
+        publicPath: process.env.BASE_URL,
+        d3: null,
+        svg:  null,
+        svgGroup: null,
+        dialogData: null,
+        showDialog: false,
+        cardHeader: null,
+        cardTitle: null,
+        cardText: null,
+        cardImageSource: null,
+        cardAltText: null
+      }
+    },
+    mounted(){
+      this.d3 = Object.assign(d3Base);
+
+      // create svg that will hold chart
+      this.svg = this.d3.select("#svg-container").append("svg")
+        .attr("viewBox", "0 0 600 400")
+        .attr("preserveAspectRatio", "xMidYMid meet")
+        .attr("width", '100%')
+        .attr("height", '100%')
+
```

Comment:
This makes sense as the original application it was used in, but would it be more general to apply this to plain text? That would make this example a little simpler, or you could provide ways in the example.

Reply:
Add dynamic and singular buttons as examples